Week3/homework/3-step3.js

```diff
@@ -3,6 +3,13 @@
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
+  if (num < 1) {
+    return '';
+  }
+  let myStr = str;
+  for (let i = 1; i < num; i++) {
+    str += myStr;
+  }
```

Comment:
Umm.. I know it has not been introduced to the class in a detailed level but try remembering this. Always stride for having your functions pure. You can try a few things to make sure you adhere to it. For example, avoid using variables from outside a function's scope and also avoid manipulating a function's parameters. In this case, let's run the loop from 0 and declare `myStr` to be an empty string and then assign it further values and return `myStr` instead of `str`. Otherwise, the task is fine. 🙂